document/core/conf.py

```diff
@@ -62,8 +62,12 @@
 title = u'WebAssembly Specification'
 copyright = u'2017, WebAssembly Community Group'
 author = u'WebAssembly Community Group'
-editor = u'Andreas Rossberg (editor)'
+editor_name = u'Andreas Rossberg'
+editor = editor_name + u' (editor)'
```

Comment:
Nit: rename editor_name to editor and inline the "(editor)" where needed? (I see only one occurrence anyway)